src/helpers/index.js

```diff
@@ -94,3 +94,4 @@ export { useDetectClickOutside } from './Utils/useDetectClickOutside';
 export { usePrevious } from './Utils/usePrevious';
 export { usePagination } from './Utils/usePagination';
 export useUndoManager from './UndoManager/useUndoManager';
+export Popup from './Popup/Popup';
```

Comment:
I thought about it after the team meeting today and I'm not a fan of having an actual component in the helpers folder. I would personally prefer moving it to the components folder. Just my 2 cents on this.